### DISCUSSION TITLE
How to use single roles in guard api and web (all guard) Without make 1 name with different guard_name

### DISCUSSION THREAD

**Discussion by rickyanwar** [spatie/laravel-permission#2441] [Q&A]
I want create role , to all guard 
Web / api, with one data not create for different guard_name it possible? 
mean this roles can use both api/web

I was try to make guard_name to Asterix,l in roles, but when assigned to user can't, 

if I create roles with guard_name = web when assigned to user via api, it not found (role name)

**Comment by erikn69** [comment]
https://github.com/spatie/laravel-permission/discussions/2421#discussioncomment-5785479

**Reply by rickyanwar** [reply]
Web and api have same function, 
I used spatie for manage the role and permission, but how I used singel role but can used in guard api and web without create both
